Source/WebKit/Resources/SandboxProfiles/ios/com.apple.WebKit.GPU.sb.in

```diff
@@ -585,7 +588,9 @@
         "kern.willshutdown" ;; <rdar://122511261>
         "machdep.ptrauth_enabled"
         "vm.footprint_suspend"
-        "vm.malloc_ranges")) ;; <rdar://problem/105161083>
+        "vm.malloc_ranges") ;; <rdar://problem/105161083>
+    (sysctl-name-prefix "hw.perflevel") ;; <rdar://problem/76782530>
```

Comment:
I think it would be good to void prefix notation, since this can potentially allow more than needed.

Reply:
That's a good point. However, I think we should use the prefix notation for these `hw.perflevel` things since we are already using this syntax for the WebContent sandbox on all platforms (plus the GPU Process on macOS), and that was based on the rules SEAR approved for the other OS sandboxes.